13_biblioteki/13_6_TemplatesForSolutions.tex

```diff
@@ -0,0 +1,29 @@
+% !TEX encoding = UTF-8 Unicode
```

Comment:
All of that stuff you moved doeas belong in the bibliography - good job moving it ;)